person.rb

```diff
@@ -27,4 +28,9 @@ def can_use_services?
   def correct_name
     @name
   end
+
+  def add_rental(book, date)
+    rental = Rental.new(book, date, self)
+    @rentals << rental
+  end
```

Comment:
Good job by creating add_rental method but  @rentals << rental is already done in Rental initializer so no need to redo it here